.ci/setup_env.sh

```diff
@@ -108,21 +108,21 @@ eval `luarocks path`
 # -------------------------------------
 # Install ccm & setup Cassandra cluster
 # -------------------------------------
-if [[ "$TEST_SUITE" != "unit" ]] && [[ "$TEST_SUITE" != "lint" ]]; then
-  echo "Installing ccm and setting up Cassandra cluster..."
-  pip install --user PyYAML six ccm &> build.log || (cat build.log && exit 1)
-  ccm create test -v $CASSANDRA -n 1 -d
```

Comment:
I'll make a card to do multi-cassandra testing. I swapped ccm out for docker because it was a single cassandra node and the docker one ran quicker / more reliably in my experience